playbook.yaml

```diff
@@ -157,6 +157,8 @@
       creates: '{{ansible_env.HOME}}/fabricant-prod/.secrets/gf_admin_password.txt'
   - name: Setup Label Studio
     import_tasks: label_studio.yaml
+  - name: Setup authentik
```

Comment:
To confim, we cannot use authentik with label studio? If so dismiss, otherwise we may need to swap lines 158 and 160, since label studio would depend on it

Reply:
Label Studio is only compatible if buy enterprise.  These lines don't start the services, so order does not matter here.